lib/brainstem/presenter_collection.rb

```diff
@@ -226,6 +226,11 @@ def handle_only(scope, only)
 
     def run_filters(scope, options)
       extracted_filters = extract_filters(options)
+
+      params = extracted_filters.each.with_object({}) do |(key, val), hash|
+        hash[key] = val[:arg]
```

Comment:
My thought was that you might want to know about the existence of other filters (including default filters), and not just their args. Although I can't think of a specific use case for that. We can check for `val[:arg].nil?` if you think that would never be needed.